modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+
+| `conflictLogging`
+| The main setting which when non-empty and valid enables logging of conflicts.
+
+For more information, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflictlogging-key[conflictLogging Key].
+|
+
+| `cLogSetTimeoutMs`
+| Sets the timeout for SetMeta operation for conflict writing.
+| 5000ms
+
+| `cLogPoolGetTimeoutMs`
+| Timeout for getting connection from connection pool
+| 5000ms
+
+| `cLogNetworkRetryCount`
+| Number of attempts for a single doc when there is a network error
+| 5
+
+| `cLogNetworkRetryIntervalMs`
+| Time interval to sleep between attempts to write the conflict doc on network error
+| 2000ms
+
+| `cLogWorkerCount`
+| Number of workers used by the conflict logger, which actually write conflict documents.
+| 20
+
+| `cLogQueueCapacity`
+| Queue capacity for the conflict logger.
+| 6000
+
+| `cLogMaxErrorCount`
+| Max number of errors encountered by conflict logger in a “Error Window” after which the conflict logger will be hibernated.
+
+For information about hibernation, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#reliability-components-logger[Reliability Components].
+| 10
+
+| `cLogErrorTimeWindowMs`
+| This is the time window in which the number of errors in the conflict logger will be checked.
+| 120000ms
+
+| `cLogReattemptDurationMs`
+| This is the duration for which the conflict logger will be hibernated and after this the logger will be active again.
+
+For information about hibernation, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#reliability-components-logger[Reliability Components].
+| 600000ms
+|===
+
+[#disable-conflict-logging]
+== Disable Conflict Logging
+
+This section explains disabling XDCR Conflict Logging feature.
+
+* *HLV*: The bucket property `enableCrossClusterVersioning` cannot be disabled once it is enabled.
+For more information, see xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[XDCR enableCrossClusterVersioning].
+Enabling HLV depends on `enableCrossClusterVersioning`, so HLV also cannot be disabled after it is enabled.
+
+* *Setting the replication setting off*: Set the `conflictLogging` parameter to `{}` or `{ "disabled": true }` because an empty JSON object value must be specified to disable the conflict logging capability of XDCR.
++
+For configuration information, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#use-conflictlogging-and-loggingrules[Use conflictLogging and loggingRules].
+When it is disabled, any conflict documents previously created by XDCR in the conflict collections remain in place.
+
+* *Deleting a Replication*: When you delete a replication, conflict record documents in the conflict collection remain until you manually delete them.
+As you can change the conflict collection multiple times during the lifecycle of a replication, XDCR does not automatically clean up previously created conflict records.
```

Comment:
XDCR does not clean up -- automatically or any other way.  

Once XDCR writes the conflict record documents into the conflict collection, you manage the documents.  Since you can change the conflict collection multiple times during the lifecycle of a replication, XDCR will not clean up or delete previously created conflict record documents.